drake/systems/plants/@Manipulator/Manipulator.m

```diff
@@ -92,6 +92,68 @@
 
     end
 
+    function [xdot,dxdot,ps,qs,qds,us,ts] = dynamicsSym(obj,x,u,t)
```

Comment:
I haven't moved this function in the current iteration - still trying to figure out where to put it for now, especially since it might be phased out if I can figure it out with generateGradients.